test/integration/fetch-snyk-java-call-graph-generator.test.ts

```diff
@@ -0,0 +1,21 @@
+import * as path from 'path';
```

Comment:
Why is this test necessary?

Reply:
I want something to make sure the actual sha on the config matches the JAR on the config